Source/Flow/Private/FlowAsset.cpp

```diff
@@ -189,35 +189,73 @@ void UFlowAsset::HarvestNodeConnections()
 	}
 }
 
-void UFlowAsset::AddCustomInput(const FName& InName)
+void UFlowAsset::AddCustomInput(const FName& EventName)
 {
-	if (!CustomInputs.Contains(InName))
+	if (!CustomInputs.Contains(EventName))
 	{
-		CustomInputs.Add(InName);
+		CustomInputs.Add(EventName);
 	}
 }
 
-void UFlowAsset::RemoveCustomInput(const FName& InName)
+void UFlowAsset::RemoveCustomInput(const FName& EventName)
 {
-	if (CustomInputs.Contains(InName))
+	if (CustomInputs.Contains(EventName))
 	{
-		CustomInputs.Remove(InName);
+		CustomInputs.Remove(EventName);
 	}
 }
+
+void UFlowAsset::AddCustomOutput(const FName& EventName)
+{
+	check(!CustomOutputs.Contains(EventName));
+
+	CustomOutputs.Add(EventName);
+}
+
+void UFlowAsset::RemoveCustomOutput(const FName& EventName)
+{
+	check(CustomOutputs.Contains(EventName));
+
+	CustomOutputs.Remove(EventName);
+}
 #endif // WITH_EDITOR
 
+UFlowNode_CustomInput* UFlowAsset::TryFindCustomInputNodeByEventName(const FName& EventName) const
```

Comment:
New (public) function to find a custom input by name.  

We use it in our code to look to see if the flowgraph has specific inputs, and check if they have connected nodes, before attempting to call them.